lbrynet/core/client/ConnectionManager.py

```diff
@@ -44,17 +46,18 @@ def _get_log_name(self):
             out = 'Connection Manager '+self.downloader.blob_hash
         return out
 
-    def start(self):
-        log.debug("%s starting", self._get_log_name())
-        from twisted.internet import reactor
-
+    def _start(self):
```

Comment:
this function name doesn't match what it does - not sure what a better name would be, but worth thinking about.